lib/Reconciler.lua

```diff
@@ -507,12 +509,31 @@ function Reconciler._setRbxProp(rbx, key, value, element)
 			)
 
 			error(message, 0)
+		elseif GlobalConfig.getValue("logAllMutations") then
+			local message = ("<TRACE> Setting %s on %s of class %s to %s"):format(
+				key,
+				rbx:GetFullName(),
+				rbx.ClassName,
+				tostring(value)
+			)
+
+			Reconciler._traceFunction(message)
```

Comment:
I think this condition is kind of weird.

We should move it to be above the `not success` block instead of attaching an `elseif` to a block that has a negative condition. It makes sense to trace mutations even if they fail!

Reply:
Yeah, that's a weird conditional. Fixed now!